src/store/getters.ts

```diff
@@ -225,9 +519,33 @@ export const getEntityTypesXPRO = (state: StateIF): Array<EntityI> => {
   }
 }
 
+export const getShowXproJurisdiction = (state: StateIF): boolean => {
+  if (getLocation(state) !== 'BC') {
+    return true
+  }
+  if (getRequestActionCd(state) === 'MVE') {
+    return true
+  }
+  return false
+}
+
+export const getXproRequestTypeCd = (state: StateIF): string => {
+  if (getIsAssumedName(state)) {
+    switch (getEntityTypeCd(state)) {
+      case 'RLC':
+        return 'AL'
+      case 'XCR':
+        return 'AS'
+      default:
+        return ''
+    }
+  }
+  return ''
+}
+
 /** Get entity type options. */
-export const getEntityTypeOptions = (state: StateIF) => {
-  let bcOptions: SelectOptionsI[] = getEntityTypesBC(state).filter(x => {
+export const getEntityTypeOptions = (state: StateIF): Array<EntityI> => {
+  let bcOptions: SelectOptionsI[] = getEntityTypesBC(state)?.filter(x => {
```

Comment:
Did you add this because you found an error?

The only problem is this might be masking an error, and we should fix the root cause instead.

Reply:
In a couple places, I added some "safety checks" like this, but I also put in a console.error() to report failures (and saw some, too). So there are Sentry errors that I can go back and look at and try to fix the root cause.